tests/uberscan.test.sh

```
@@ -15,172 +22,83 @@ ck "has allowed-tools" "grep -q '^allowed-tools:' '$CMD'"
-# with rate-limit-budget-insufficient). The SKILL must NOT re-grow a copy.
-ck "no pre-dispatch gh rate_limit probe remains in the SKILL" "[ \$(grep -c 'gh api rate_limit' '$SKILL') -eq 0 ]"
-ck "no RATE_OK gating remains in the SKILL" "[ \$(grep -c 'RATE_OK' '$SKILL') -eq 0 ]"
-ck "CB6 delegation to findings-to-issues Step 2 documented" "grep -q 'findings-to-issues.md Step 2' '$SKILL'"
+echo "== U8: chunk.py CLI contract (the relay must use REAL flags — guards the --out blocker class) =="
+# The Workflow harness STUBS agents, so the CLI strings inside scan-fleet.js relay
+# prompts are never executed by the behavioral fixtures. A fabricated flag (the
+# `--out` class — chunk.py has no --out, it writes JSON to stdout) would ship
+# silently and break EVERY live run. Lock chunk.py's real interface here + assert
+# the relays redirect stdout instead of passing --out.
+CHUNK="$REPO_ROOT/plugins/uberdev/lib/chunk.py"
+WF="$REPO_ROOT/plugins/uberdev/skills/scan-fleet/workflow.js"
```

Shellcheck (0.11.0)

[warning] WF appears unused. Verify use (or export if used externally).

(SC2034)